src/app/item-triage/ItemTriage.tsx

```diff
@@ -270,7 +270,8 @@ function collectRelevantItemFactors(exampleItem: DimItem, allItems: DimItem[]) {
     .forEach((item: DimItem) => {
       factorCombos[exampleItem.bucket.sort as factorComboCategory].forEach((factorCombo) => {
         const combination = applyFactorCombo(item, factorCombo);
-        combinationCounts[combination] = (combinationCounts[combination] ?? 0) + 1;
+        combinationCounts[combination] ??= 0 + 1;
```

Comment:
This feels like it shouldn't have the +1

Reply:
miscommit, thank you!